CHAOSScon/2020EU/desc.md

```diff
@@ -12,6 +12,22 @@ Co-located with FOSDEM
 
 January 31, 2020
 
+## Registration
+
+Registration will open soon!
+
+### Diversity and Inclusion
+
+To encourage and increase the participation of a diverse community, we are pleased to offer waivers for registration based on diversity and need. To get a your registration waived, please contact [Georg Link](linkgeorg@gmail.com)
+
+All speakers and attendees are required to adhere to our [Event Code of Conduct](https://events.linuxfoundation.org/about/code-of-conduct/).
```

Comment:
I think it might be better if we re-order this a bit. Something like this maybe:

## Code of Conduct

All speakers and attendees ...

## Registration

Registration will open soon!

### Diversity and Inclusion

To encourage ...

Reply:
Yep, that makes sense, I will edit the order.